test/dnn_images_test.cpp

```diff
@@ -17,6 +18,7 @@ class DnnImagesTest : public ::testing::Test {
     nh_priv.getParam("image_directory", image_directory);
     object_sub = nh.subscribe("/dnn_objects", 1, &DnnImagesTest::object_callback, this);
     got_object = false;
+    got_cat = false;
```

Comment:
Can you upload/link to your test cat image? 

Put it in `test/data` if you put it in the repo?

Reply:
It's already there in `test/test_images/cat.jpg`

Reply:
Ah ok, I thought it was new and I didn't see it in the PR.